docs/fuzz-settings.md

```diff
@@ -197,35 +416,612 @@ The parameters of the `hookFn` are as follows:
+it.skip.fuzz("skipped test", (data) => {...});
+it.fuzz("this test will be fuzzed", (data) => {...});
+```
+
+In this example, the third test named `"fuzzed test"` will be fuzzed because we
+use `only`:
+
+```javascript
+it.fuzz("skipped test 1", (data) => {...});
+it.fuzz("skipped test 2", (data) => {...});
+it.only.fuzz("fuzzed test", (data) => {...});
+```
+
+**ENV:** To run in the mode other than the default mode in CLI or Jest, set the
+environment variable `JAZZER_MODE`. For example, to select the `regression` mode
+in CLI:
+
+```bash
+JAZZER_MODE=regression npx jazzer my-fuzz-file
+```
+
+To select the `fuzzing` mode in Jest:
+
+```bash
+JAZZER_MODE=fuzzing npx jest tests.fuzz.js
+```
+
+_Note:_ In Jest mode, setting `JAZZER_MODE=fuzzing` is the same as setting
+[`JAZZER_FUZZ=1`](#jazzer_fuzz--boolean).
+
+### `sync` : [boolean]
+
+Default: false
+
+Run in synchronous mode.
+
+If the code under test is fully synchronous, fuzzing will be faster in
+synchronous mode.
+
+_Note:_ don't use this option if the code under test contains asynchronous
+parts.
+
+**CLI:** To run in synchronous mode on command line, append the `--sync` flag to
+the command:
+
+```bash
+npx jazzer my-fuzz-file --sync
+```
+
+**Jest:** To run in synchronous mode in Jest mode, add the following to the
+`.jazzerjsrc.json` file:
+
+```json
+{
+	"sync": true
+}
+```
+
+**ENV:** To run in synchronous mode in CLI or Jest mode, set the environment
+variable `JAZZER_SYNC` to `true`. Here is an example for Jest:
+
+```bash
+JAZZER_SYNC=true npx jest tests.fuzz.js
+```
+
+### `timeout` : [number]
+
+Default: 5000 (milliseconds)
+
+Set fuzz test timeout in milliseconds.
+
+If a fuzz test takes longer than `timeout` to execute an input, Jazzer.js will
+save this input in a crash file.
+
+**CLI:** To set the timeout to 10000 milliseconds on command line, use:
+
+```bash
+npx jazzer my-fuzz-file --timeout=10000
+```
+
+**Jest:** To set the timeout to 10000 milliseconds in Jest mode, add the
+following to the `.jazzerjsrc.json` file:
+
+```json
+{
+	"timeout": 10000
+}
+```
+
+**ENV:** To set the timeout to 10000 milliseconds in CLI or Jest mode, set the
+environment variable `JAZZER_TIMEOUT`. Here is an example for Jest:
+
+```bash
+JAZZER_TIMEOUT=10000 npx jest tests.fuzz.js
+```
+
+### `verbose` : [boolean]
+
+Default: false
+
+Print debugging logs including:
+
+- current values for all configuration options set by CLI/Jest flags and
+  environment variables
+- hooked functions
+- functions that could not be hooked
+- functions available to hooking
+
+**CLI:** Add `--verbose` flag to the command:
+
+```bash
+npx jazzer my-fuzz-file --verbose
+```
+
+**Jest:** Add the `verbose` option to the Jazzer.js configuration file
+`.jazzerjsrc.json`:
+
+```json
+{
+	"verbose": true
+}
+```
+
+**ENV:** To set the flag for CLI and Jest mod, use the environment variable
+`JAZZER_VERBOSE`. Here is an example for CLI:
+
+```bash
+JAZZER_VERBOSE=1 npx jazzer my-fuzz-file
+```
+
 ## Bug Detectors
```

Comment:
Some here. Extract out together with the hooks section?

Reply:
Extracted into bug-detectors.md